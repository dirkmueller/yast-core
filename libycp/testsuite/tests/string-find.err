Parsed:
----------------------------------------------------------------------
"test cases for findfirstof"
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(findfirstof ("abcdefghi", "cxdv") == 2)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(findfirstof ("aaaaa", "z") == nil)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(findfirstof ("", "a") == nil)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(findfirstof ("", "") == nil)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
"test cases for findlastof"
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(findlastof ("abcdefghi", "abcefghi") == 8)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(findlastof ("abcdefghi", "z") == nil)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(findlastof ("", "abcdefghi") == nil)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(findlastof ("", "") == nil)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
"test cases for findfirstnotof"
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(findfirstnotof ("abc", "ac") == 1)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(findfirstnotof ("abc", "abc") == nil)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(findfirstnotof ("abc", "") == 0)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(findfirstnotof ("", "abcdefghi") == nil)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(findfirstnotof ("", "") == nil)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
"test cases for findlastnotof"
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(findlastnotof ("abc", "ac") == 1)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(findlastnotof ("abc", "abc") == nil)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(findlastnotof ("abc", "") == 2)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(findlastnotof ("", "abcdefghi") == nil)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(findlastnotof ("", "") == nil)
----------------------------------------------------------------------
