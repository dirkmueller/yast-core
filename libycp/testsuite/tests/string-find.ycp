
( "test cases for findfirstof" )

( findfirstof ( "abcdefghi", "cxdv" ) == 2 )
( findfirstof ( "aaaaa", "z" ) == nil )
( findfirstof ( "", "a" ) == nil )
( findfirstof ( "", "" ) == nil)


( "test cases for findlastof" )

( findlastof ( "abcdefghi", "abcefghi" ) == 8 )
( findlastof ( "abcdefghi", "z" ) == nil )
( findlastof ( "", "abcdefghi" ) == nil )
( findlastof ( "", "" ) == nil)


( "test cases for findfirstnotof" )

( findfirstnotof ( "abc", "ac" ) == 1 )
( findfirstnotof ( "abc", "abc" ) == nil )
( findfirstnotof ( "abc", "" ) == 0 )
( findfirstnotof ( "", "abcdefghi" ) == nil )
( findfirstnotof ( "", "" ) == nil )


( "test cases for findlastnotof" )

( findlastnotof ( "abc", "ac" ) == 1 )
( findlastnotof ( "abc", "abc" ) == nil )
( findlastnotof ( "abc", "" ) == 2 )
( findlastnotof ( "", "abcdefghi" ) == nil )
( findlastnotof ( "", "" ) == nil )

