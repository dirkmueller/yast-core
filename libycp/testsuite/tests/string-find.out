("test cases for findfirstof")
(true)
(true)
(true)
(true)
("test cases for findlastof")
(true)
(true)
(true)
(true)
("test cases for findfirstnotof")
(true)
(true)
(true)
(true)
(true)
("test cases for findlastnotof")
(true)
(true)
(true)
(true)
(true)
